src/main/java/frc/robot/subsystems/climber/ClimberIOTalonFX.java

```diff
@@ -0,0 +1,225 @@
+
+      this.leftMotor.getConfigurator().refresh(config);
+      config.Slot0.kP = kP.get();
+      config.Slot0.kI = kI.get();
+      config.Slot0.kD = kD.get();
+      config.Slot0.kS = kS.get();
+      config.Slot0.kV = kV.get();
+      config.Slot0.kA = kA.get();
+      motionMagicConfigs.MotionMagicExpo_kV = kVExpo.get();
+      motionMagicConfigs.MotionMagicExpo_kA = kAExpo.get();
+
+      this.leftMotor.getConfigurator().apply(config);
+      this.rightMotor.getConfigurator().apply(config);
+    }
+  }
+
+  /**
+   * Set the leftMotor position to the specified value in degrees.
+   *
+   * @param position the position to set the leftMotor to in meters
+   * @param arbitraryFeedForward the arbitrary feed forward as a percentage of maximum power
+   */
+  @Override
+  public void setLeftMotorPosition(double position) {
+    leftMotor.setControl(
+        leftPositionCurrentRequest
+            .withPosition(Conversions.degreesToFalconRotations(position, GEAR_RATIO))); // FIXME: Feed forward should be acounted for already.
+  }
+
+  @Override
+  public void setRightMotorPosition(double position) {
+    this.rightMotor.setControl(
+        rightPositionCurrentRequest
+            .withPosition(Conversions.degreesToFalconRotations(position, GEAR_RATIO)));
+  }
+
+  @Override
+  public void setLeftMotorPower(double current) {
+    leftMotor.setControl(leftSupplyAmpsRequest.withOutput(current));
+  }
+
+  @Override
+  public void setRightMotorPower(double current) {
+    rightMotor.setControl(rightSupplyAmpsRequest.withOutput(current));
+  }
+
+  @Override
+  public void setPositionZero() {
+    leftEncoder.setPosition(0);
+    rightEncoder.setPosition(0);
+  }
+
+  private void configMotors(TalonFX leftMotor, TalonFX rightMotor) {
+    this.leftMotor = new TalonFX(LEFT_MOTOR_CAN_ID, RobotConfig.getInstance().getCANBusName());
+    this.rightMotor = new TalonFX(RIGHT_MOTOR_CAN_ID, RobotConfig.getInstance().getCANBusName());
+
+    this.leftEncoder = new CoreCANcoder(LEFT_MOTOR_CAN_ID);
+    this.rightEncoder = new CoreCANcoder(RIGHT_MOTOR_CAN_ID);
+
+    TalonFXConfiguration config = new TalonFXConfiguration();
+
+    CurrentLimitsConfigs currentLimits = new CurrentLimitsConfigs();
+    currentLimits.SupplyCurrentLimit = CONTINUOUS_CURRENT_LIMIT;
+    currentLimits.StatorCurrentLimit = STATOR_CURRENT_LIMIT;
+    currentLimits.SupplyCurrentThreshold = PEAK_CURRENT_LIMIT;
+    currentLimits.SupplyTimeThreshold = PEAK_CURRENT_DURATION;
+    currentLimits.SupplyCurrentLimitEnable = true;
+    config.CurrentLimits = currentLimits;
+
+    config.Feedback.SensorToMechanismRatio = GEAR_RATIO;
+    config.MotorOutput.Inverted =
+        MOTOR_INVERTED ? InvertedValue.Clockwise_Positive : InvertedValue.CounterClockwise_Positive;
+    config.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+    config.Slot0.kP = kP.get();
+    config.Slot0.kI = kI.get();
+    config.Slot0.kD = kD.get();
```

Comment:
need to set kS, kV, kA, and kG as well

also set kV and kA for the motion magic